File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/10300_:::_Ecological_Premium/Main.c
#include "stdio.h"

int test=1;

int main()
{
	if(test) freopen("in.txt","r",stdin);

	int casos, granjas, rta, size, environment, noUso;
	scanf("%d",&casos);
	while(casos--)
	{
		scanf("%d",&granjas);
		rta=0;
		while(granjas--)
		{
			scanf("%d %d %d",&size, &noUso, &environment);
			rta+=(size*environment);
		}
		printf("%d\n",rta);
	}
		
	return 0;
}

File: Capitulo_01/01_Getting_Started:_The_Easy_Problems/02_Easy/10300_:::_Ecological_Premium/Main.java
import java.io.BufferedReader;
import java.io.File;
import java.io.FileReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main
{
	public static void main(String[] args) throws IOException
	{
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		//-----------------------------------------------------------
		int casos=Integer.parseInt(br.readLine());

		while(casos--!=0)
		{
			int granjas=Integer.parseInt(br.readLine()), rta=0;
			while(granjas--!=0)
			{
				String[] aux=br.readLine().split(" ");
				int size=Integer.parseInt(aux[0]);
				int environment=Integer.parseInt(aux[2]);

				rta+=(size*environment);
			}
			System.out.println(rta);
		}		
		//-----------------------------------------------------------
		br.close();
	}
}
